src/uniparc/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,291 @@
+                            pathname: LocationToPath[Location.UniParcResults],
+                            search: stringifyQuery({
+                              query: '*',
+                              facets: `${FacetsEnum.Database}:${value}`,
+                            }),
+                          }}
+                        >
+                          {label || value}
+                        </Link>
+                      </td>
+                      <td className={styles.end}>
+                        <LongNumber>{count}</LongNumber>
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+            ))}
+          </div>
+
+          <Button
+            variant="tertiary"
+            onClick={() => setExpanded((expanded) => !expanded)}
+          >
+            {expanded ? 'View less' : 'Explore more'} databases
+          </Button>
+        </section>
+
+        {/* Text */}
+        <section
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            `uniprot-grid-cell--medium-span-${expanded ? '9' : '5'}`
+          )}
+        >
+          <h2>Searching the Database</h2>
+          <br />
+          <p>
+            UniParc is available for text- and sequence-based searches.
+            Performing a similarity search against UniParc is equivalent to
+            performing the same search against all databases cross-referenced in
+            UniParc, as UniParc contains all proteins from its source databases.
+          </p>
+          <p>
+            Sequence similarity searches can be done using{' '}
+            <Link
+              to={{
+                pathname: LocationToPath[Location.Blast],
+                search: stringifyQuery({
+                  database: 'uniparc',
+                }),
+              }}
+            >
+              BLAST
+            </Link>
+          </p>
+        </section>
+
+        {/* Downloads */}
+        <section className="uniprot-grid-cell--small-span-12 uniprot-grid-cell--medium-span-3">
+          <h2>Download</h2>
+          <div className={styles.download}>
+            <br />
+            <p>
+              UniParc sequence archive
+              <br />
+              <ExternalLink url="https://ftp.uniprot.org/pub/databases/uniprot/current_release/uniparc/xml/all/">
+                xml
+              </ExternalLink>
+              <ExternalLink url="https://ftp.uniprot.org/pub/databases/uniprot/current_release/uniparc/fasta/active/">
+                fasta
+              </ExternalLink>
+              <ExternalLink url="https://ftp.uniprot.org/pub/databases/uniprot/current_release/uniparc/README">
+                readme
+              </ExternalLink>
+            </p>
+            <p>
+              XML
+              <br />
+              <ExternalLink url="https://ftp.uniprot.org/pub/databases/uniprot/current_release/uniparc/uniparc.xsd">
+                xml schema
```

Comment:
I'm not sure if the inclusion of the XML schema is needed here? At first glance it confused me a bit as I was under the impression that it was the actual XML file download which is actually listed above. We have quite a lot of space so not sure what you think about a traditional list? Eg

```
UniParc sequence archive
- xml (schema)
- fasta
- readme

Explore more FTP
```

I know it breaks with the uniprotkb landing page but for me it organizes the links a bit better.